ui/vite.config.ts

```diff
@@ -42,10 +42,9 @@ export default defineConfig({
   ],
   server: {
     proxy: {
-      '/api': {
+      '': {
```

Comment:
```suggestion
      '/akuity.io.kargo.service.v1alpha1.KargoService': {
```

Let's apply this change to make local development work